src/main/java/dansplugins/factionsystem/utils/ArgumentParser.java

```diff
@@ -31,7 +31,7 @@ public String createStringFromFirstArgOnwards(String[] args) {
     public ArrayList<String> getArgumentsInsideSingleQuotes(String[] args) {
         ArrayList<String> toReturn = new ArrayList<>();
 
-        String argumentString = ArgumentParser.getInstance().createStringFromFirstArgOnwards(args);
+        String argumentString = String.join(" ", args);
```

Comment:
I just wanna mention that I did triple check the usage of this method and the only times it is used is by methods which I have now modified to use the new command implementation, making the "crecreateStringFromFirstArgOnwards" method obsolete and I remove the unrequired argument from the String array.

- Callum

Reply:
Sounds good!